lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3303,9 +3303,17 @@ def test_learner_redirect(self):
             assert self.client.get(lms_url).url == mfe_url
 
     def test_staff_no_redirect(self):
-        # global staff will never be redirected
         lms_url, mfe_url = self._get_urls()
 
+        # course staff will not redirect
+        course_staff = UserFactory.create(is_staff=True)
```

Comment:
Doesn't this create a global staff (a user with `is_staff=True`), as opposed to a course staff that's been added to the appropriate group?

Reply:
Ah, you're right. I fixed it.